packages/paste-design-tokens/tokens/global/border-color.yml

```diff
@@ -62,6 +62,9 @@ props:
   color-border-warning-lightest:
     value: "{!palette-orange-10}"
     comment: Lightest warning border color
+  color-border-error-darker:
+    value: "{!palette-red-80}"
+    comment: Darker Error border color
```

Comment:
This should be dark rather than darker in all themes + the form field error state.